packages/engine/src/worker/runner/python/batch.py

```diff
@@ -87,8 +87,10 @@ def load_dataset(batch_id):
     (_, _, header_offset, header_size, _, _, data_offset, data_size) = load_markers(mem)
 
     # The header has the shortname of the dataset
-    header_buf = mem[header_offset: header_offset + header_size]
-    dataset_name = str(header_buf.to_pybytes().decode('utf-8'))
+    n_metaversion_bytes = 8  # Memory u32 + batch u32 version
```

Comment:
I feel like we should store this more obvious? Perhaps as a global at the top?

Reply:
Replaced by `2 * mem::size_of::<u32>()`

Reply:
Whoops, only changed the occurrences in Rust. Made this a variable at the top and changed it.

Reply:
So are you saying to ignore this until the Python branch?

Reply:
Yes, my code is literally touching this exact section because there were some adjustments needed. My commits are ready to be pushed as soon as this PR is merged.